src/hii_road.py

```diff
@@ -1,358 +1,172 @@
-            .add(highway_bus_guideway.multiply(self.highway_bus_guideway_weighting))
-            .add(highway_cycleway.multiply(self.highway_cycleway_weighting))
-            .add(highway_elevator.multiply(self.highway_elevator_weighting))
-            .add(highway_escape.multiply(self.highway_escape_weighting))
-            .add(highway_footway.multiply(self.highway_footway_weighting))
-            .add(highway_living_street.multiply(self.highway_living_street_weighting))
-            .add(highway_mini_roundabout.multiply(self.highway_mini_roundabout_weighting))
-            .add(highway_motorway.multiply(self.highway_motorway_weighting))
-            .add(highway_motorway_link.multiply(self.highway_motorway_link_weighting))
-            .add(highway_path.multiply(self.highway_path_weighting))
-            .add(highway_pedestrian.multiply(self.highway_pedestrian_weighting))
-            .add(highway_primary.multiply(self.highway_primary_weighting))
-            .add(highway_primary_link.multiply(self.highway_primary_link_weighting))
-            .add(highway_raceway.multiply(self.highway_raceway_weighting))
-            .add(highway_rest_area.multiply(self.highway_rest_area_weighting))
-            .add(highway_road.multiply(self.highway_road_weighting))
-            .add(highway_secondary.multiply(self.highway_secondary_weighting))
-            .add(highway_secondary_link.multiply(self.highway_secondary_link_weighting))
-            .add(highway_service.multiply(self.highway_service_weighting))
-            .add(highway_steps.multiply(self.highway_steps_weighting))
-            .add(highway_tertiary.multiply(self.highway_tertiary_weighting))
-            .add(highway_tertiary_link.multiply(self.highway_tertiary_link_weighting))
-            .add(highway_track.multiply(self.highway_track_weighting))
-            .add(highway_trunk.multiply(self.highway_trunk_weighting))
-            .add(highway_trunk_link.multiply(self.highway_trunk_link_weighting))
-            .add(highway_turning_circle.multiply(self.highway_turning_circle_weighting))
-            .add(highway_unclassified.multiply(self.highway_unclassified_weighting))
-            .add(ee.Image(self.inputs["groads_additions"]["ee_path"]).multiply(self.highway_groads_weighting))
-        ).multiply(2)
-
-
-        roads_bool = highway_total.gt(0)
-
-        roads_indirect = roads_bool.eq(0)\
-            .cumulativeCost(roads_bool, 15000)\
-            .reproject(crs=self.crs, scale=self.scale)\
-            .multiply(self.DECAY_CONSTANT)\
-            .exp()\
-            .multiply(self.INDIRECT_INFLUENCE)\
-            .unmask(0)
-
-        roads_total = highway_total.add(roads_indirect).updateMask(watermask)
-
         self.export_image_ee(
-            roads_total,
-            "{}/{}".format(self.ee_driverdir, "aois/" + self.realm),
-            
+            road_driver,
+            f"driver/roads",
         )
 
+    # Check inputs is not called if date is before OSM_START
```

Comment:
Probably can lose this comment... and I don't like skipping `check_inputs` but I don't see a better way and in this particular task with these particular inputs it makes sense.

Reply:
I agree, it seems to defeat the purpose of check_inputs.  but I also can't think of a better way at the moment.